app/policies/user_policy.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class UserPolicy < AdminPolicy
+  %w[index? destroy?].each do |action|
+    define_method(action) do
+      user_admin?
+    end
+  end
```

Comment:
```
alias_method :index?, :user_admin?
alias_method :destroy?, :user_admin?
```

Reply:
I use this structure
  def index?
    user_admin?
  end

  def edit?
    user_admin? || current_user.id == model.id
  end

  def destroy?
    user_admin?
  end